bundles/org.openhab.binding.fineoffsetweatherstation/src/main/java/org/openhab/binding/fineoffsetweatherstation/internal/FineOffsetWeatherStationBindingConstants.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.fineoffsetweatherstation.internal;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+import org.openhab.core.thing.type.ChannelTypeUID;
+
+/**
+ * The {@link FineOffsetWeatherStationBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Andreas Berger - Initial contribution
+ */
+@NonNullByDefault
+public class FineOffsetWeatherStationBindingConstants {
+
+    public static final String BINDING_ID = "fineoffsetweatherstation";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_GATEWAY = new ThingTypeUID(BINDING_ID, "gateway");
+    public static final ThingTypeUID THING_TYPE_SENSOR = new ThingTypeUID(BINDING_ID, "sensor");
+
+    public static final ChannelTypeUID CHANNEL_TEMPERATURE = new ChannelTypeUID(BINDING_ID, "temperature");
```

Comment:
Sorry for such late finding, but now realize that the names of these constants has caused some confusion for me: Can you change the prefix to `CHANNEL_TYPE` as they refer to channel type id's, not channel id's?